src/book/02-system/01-foundations/02-build-system.mdx

```diff
@@ -344,6 +345,16 @@ The `nuc7i7bnh` target should be used for building and running code on the robot
 
 The `target` command will download (or, if needed, build) the Docker image for the specified target and then mark that target as active.
 
+##### Doc
```

Comment:
Just spit balling, but would something like `./b extract-reactions` or similar be a good alternative. That way we can keep the `./b doc` to be something else, like extract READMEs and comments from modules?